src/app/components/SubjectHeading/AdditionalSubjectHeadingsButton.jsx

```diff
@@ -0,0 +1,48 @@
+/* globals document */
+import React from 'react';
+import axios from 'axios';
+import { Link } from 'react-router';
+import PropTypes from 'prop-types';
+
+class AdditionalSubjectHeadingsButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onClick = this.onClick.bind(this);
+  }
+
+  onClick() {
+    this.props.updateParent(this);
+  }
+
+  render() {
+    const indentation = this.props.indentation;
+    const previous = this.props.button === 'previous';
+
+    return (
+      <li>
+        <ul className="subjectHeadingRow" >
+          <li>
+            <ul className="subjectHeadingLabelAndToggle">
+              <li onClick={this.toggleOpen} className="subjectHeadingToggle" style={{'paddingLeft': `${20*indentation}px`}}></li>
+              <li onClick={this.onClick} className="subjectHeadingLabel seeMoreButton" style={{"paddingLeft":`${20*indentation}px`}}>
+                {`${previous ? '↑' : '↓'} See more`}
+                {previous ? null : <br /> }
+                {previous ? null : '...'}
```

Comment:
Small formatting concern - "See more" and "..." are on separate lines. I don't think this is necessary.

Reply:
This will have to be restyled, the idea is that the ellipses stand in for the missing subject headings

Reply:
In that case, maybe the ellipses just shouldn't be part of the `<a href></a>`?